### PR TITLE
TIG-3323 Add documentation for workloads owned by service arch

### PR DESCRIPTION
**Pull request by jasonjhchan**

### CODE REVIEW

**Review by aggrand — APPROVED** [review]
LGTM w.r.t. using genny syntax correctly but I can't comment on the workloads or usefulness of the descriptions for the relevant stakeholders, someone on service arch or maybe @allenwux would have better comments on it

File: src/workloads/networking/SecondaryAllowed.yml [hunk 1]
```diff
@@ -1,7 +1,10 @@
 SchemaVersion: 2018-07-01
 Owner: "@mongodb/service-architecture"
 Description: |
-  TODO: TIG-3323
+  This workload runs find operations with 'secondary' readPreference and 'local' readConcern. There
+  are 100 threads allocated to run this operation, with a rate limit of 1 thread acting per 1ms.
```
Comment (BlakeIsBlake) [inline src/workloads/networking/SecondaryAllowed.yml:5]: /s/this operation/the find operations

File: src/workloads/networking/ServiceArchitectureWorkloads.yml [hunk 2]
```diff
@@ -1,7 +1,10 @@
 SchemaVersion: 2018-07-01
 Owner: "@mongodb/service-architecture"
 Description: |
-  TODO: TIG-3323
+  This workload runs find operations with 'primary' readPreference and 'local' readConcern. There
+  are 100 threads allocated to run this operation, with a rate limit of 1 thread acting per 1ms.
```
Comment (BlakeIsBlake) [inline src/workloads/networking/ServiceArchitectureWorkloads.yml:5]: /s/this operation/the find operations

File: src/workloads/networking/SecondaryAllowed.yml (hunk 1 above)
Comment (BlakeIsBlake) [reply src/workloads/networking/SecondaryAllowed.yml:5]: 1 microsecond, not millisecond

File: src/workloads/networking/ServiceArchitectureWorkloads.yml (hunk 2 above)
Comment (BlakeIsBlake) [reply src/workloads/networking/ServiceArchitectureWorkloads.yml:5]: 1 microsecond not millisecond

**Review by BlakeIsBlake — CHANGES_REQUESTED** [review]

File: src/workloads/networking/SecondaryAllowed.yml (hunk 1 above)
Comment (jasonjhchan) [reply src/workloads/networking/SecondaryAllowed.yml]: whoops nice catch

File: src/workloads/networking/ServiceArchitectureWorkloads.yml (hunk 2 above)
Comment (jasonjhchan) [reply src/workloads/networking/ServiceArchitectureWorkloads.yml]: done

**Review by jasonjhchan — COMMENTED** [review]
addressed comments

**Review by BlakeIsBlake — APPROVED** [review]

**Comment by jasonjhchan** [comment]
evergreen merge

**Pull request merged**